noxfile.py

```diff
@@ -46,5 +46,5 @@ def pylint(session):
 
 @nox.session
 def lint_flake8(session):
-    session.install("flake8", "flake8-black", "flake8-isort")
```

Comment:
I have removed flake8-isort from the noxfile and it seems to have solved the build issue. I have been looking about this issue. And apparently there has always been trouble with isort, flake and pylint compatibility. 
Still, there has to be another way around, I tried to pin the isort version but it wasn't working either.

This is the same problem i was having: [`Link`](https://github.com/PyCQA/isort/issues/1273)